include/dlt/dlt_user.h.in

```diff
@@ -538,6 +538,29 @@ DltReturnValue dlt_user_log_write_utf8_string(DltContextData *log, const char *t
  */
 DltReturnValue dlt_user_log_write_sized_utf8_string(DltContextData *log, const char *text, uint16_t length);
 
+/**
+ * Write a constant null terminated UTF8 string into a DLT log message.
+ * In non verbose mode DLT parameter will not be sent at all.
+ * dlt_user_log_write_start has to be called before adding any attributes to the log message.
+ * Finish sending log message by calling dlt_user_log_write_finish.
+ * @param log pointer to an object containing information about logging context data
+ * @param text pointer to the parameter written into log message containing null termination.
+ * @return Value from DltReturnValue enum
+ */
+DltReturnValue dlt_user_log_write_constant_utf8_string(DltContextData *log, const char *text);
```

Comment:
Can you add new macros for added functions as well?

Reply:
OK